cargo-concordium/src/build.rs

```diff
@@ -156,10 +156,7 @@ fn create_archive(
     tar.mode(tar::HeaderMode::Deterministic);
     // Ignore files that are ignored by Git.
     let files = ignore::WalkBuilder::new(package_root_path)
-        .git_global(true)
-        .git_ignore(true)
-        .parents(true)
-        .hidden(true)
+        .standard_filters(true)
```

Comment:
Not that I am against this, but why the change?

This should also be reflected in the changelog

Reply:
Motivation was that it seemed like it was what the current code was trying to do. But given that this is affects the changelog, I would rather keep this as is.